server/routes/components.test.ts

```diff
@@ -265,14 +266,15 @@ function expectEnvironmentScreenToBeFilled(
     }[]
     monitor?: boolean
     active_agencies?: unknown
+    swagger_docs?: string
   },
 ) {
   expect($('td[data-test="name"]').text()).toBe(devEnvironment.name)
   expect($('td[data-test="type"]').text()).toBe(devEnvironment.type)
   expect($('a[data-test="url"]').attr('href')).toBe(devEnvironment.url)
   expect($('a[data-test="url"]').text()).toBe(devEnvironment.url)
   expect($('a[data-test="api"]').length).toBeGreaterThan(0)
-  expect($('a[data-test="api"]').attr('href')).toBe(`${devEnvironment.url}/swagger-ui/index.html`)
+  expect($('a[data-test="api"]').attr('href')).toBe(`${devEnvironment.url}${devEnvironment.swagger_docs}`)
```

Comment:
Assume will be failing because it doesn't exist on every environment so comes out as undefined rather than an empty string.